### PR TITLE
Faculty bios delete ces

### PR DESCRIPTION
**Pull request by c-schwarze**

### CODE REVIEW

File: tinker/templates/faculty-bios/form.html [hunk 1]
```diff
@@ -246,6 +269,73 @@ <h3 id="{{ field.id }}">{{ field.label }}</h3>
                 return fieldset_node;
             }
 
+            function addHandlerDegree(id) {
+                $("#degree-delete" + id).click(function () {
+                    id = $(this).data("id");
+                    var list = $(".degree-fieldset");
+                    for (var i = id; i < list.length; i++) {
```
[the remark this answers is not in the document]
Comment (c-schwarze) [reply tinker/templates/faculty-bios/form.html]: I don't think that is true. If there were 2 degrees [1,2]. The length is 2, so it only gets run once.

File: tinker/templates/faculty-bios/form.html [hunk 2]
```diff
@@ -246,6 +269,73 @@ <h3 id="{{ field.id }}">{{ field.label }}</h3>
...
+                    list.last().remove();
+                    checkNumDegrees();
+                })
+            };
+
+            function addHandlerJobTitle(id) {
+                $("#job-title-delete" + id).click(function () {
+                    id = $(this).data("id");
+                    var list = $(".new-job-fieldset");
+                    for (var i = id; i < list.length; i++) {
```
[the remark this answers is not in the document]
Comment (c-schwarze) [reply tinker/templates/faculty-bios/form.html]: Same comment as above

File: tinker/templates/faculty-bios/form.html [hunk 3]
```diff
@@ -246,6 +269,73 @@ <h3 id="{{ field.id }}">{{ field.label }}</h3>
                 return fieldset_node;
             }
```
[the remark this answers is not in the document]
Comment (c-schwarze) [reply tinker/templates/faculty-bios/form.html]: Yeah I recognize that as an option. Although this method has worked quite simply and keeps id's intact.

File: tinker/templates/faculty-bios/form.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (phg49389) [reply tinker/templates/faculty-bios/form.html]: Oh, the ids for the forms are 1-indexed? I hadn't caught that.

[the remark this answers is not in the document]
Comment (c-schwarze) [reply tinker/templates/faculty-bios/form.html]: 👍  yeah its so it can match up with the headings. "Degree 1" has id 1.

**Pull request merged**